aries-site/src/pages/foundation/color.mdx

```diff
@@ -0,0 +1,260 @@
+
+Color is a key way we express our brand. Using color on
+interface elements, how typography is presented, displaying data
+visualizations, and adjusting context of the experience to light
+and dark are just some of the ways we illustrate inclusive
+design for our app and web based experiences.
+
+### Brand Color
+            
+HPE green symbolizes growth and vitality, commands attention and
+inspires action. Our green is an essential part of our brand
+identity and should be evident in every communication wherever
+possible. Always use the custom color specifications seen here
+to maintain consistency across channels and media.
+
+<BrandColor />
+
+### Core Palette
+
+HPE Core colors provide a set of swatches that compliment HPE
+Green and help establish the vibrant experience HPE embraces. We
+use "emphasis" as a way to show the importance and use the most
+vibrant hue on light or dark contexts. To maintain accessibility
+requirements, it's important to be mindful of how the aspects of
+the color library are used in conjunction with the Core colors.
+
+<CorePalette />
+
+### Light Palette
+              
+The light color palette provides a set of swatches which
+compliment HPE Core colors for use in combination with dark text
+to maintain color accessibility in your experience. Use these
+colors only when implementing a light theme.
+
+<LightPalette />
+
+### Dark Palette
+
+The dark color palette provides a set of swatches which
+compliment HPE Core colors for use in combination with light
+text to maintain color accessibility in your experience. Use
+these colors only when implementing a dark theme.
+
+<DarkPalette />
+
+## Green Color Accessibility
+              
+The HPE Design System green <ColorSwatch background="green" /> ensures compliance with WCAG 2.1
+standards for distinguishable text and color ([see success criterion 1.4.1 and 1.4.3](https://www.w3.org/TR/WCAG21/#distinguishable)). 
+With accessibility as one of its core principles, the HPE
+Design System focused on keeping accessibilty research and
+guidelines at the center of its color selection process.
+
+See details below on how to remain compliant in both light and
+dark theme modes.
+              
+### Light Mode
+              
+AAA compliance is met for text on a light mode green background
+when using `text-strong` <ColorSwatch background={{color: "text-strong", dark: false}} /> color. This is ensured for text of size
+18px and above.
+
+If a font size is below 18px, AAA compliance will fail with
+default font weight on an HPE `green` <ColorSwatch background={{color: "green", dark: false}} /> background. In this case,
+apply `weight="bold"` to meet AAA compliance.
+
+<LightModeContrast />
+
+### Dark Mode
+
+AA compliance is met for text on a dark mode `green` <ColorSwatch background={{color: "green", dark: true}} /> background
+when using `text-strong` <ColorSwatch background={{color: "text-strong", dark: true}} /> color. To meet AAA compliance, use text
```

Comment:
got some reason we're not getting correct code highlighting on `text-strong`

Reply:
Yeah it was being real fussy. Not sure what is going on there. Same thing is implemented elsewhere and renders just fine.